### PR TITLE
Initial version of cryptctl setup script.

### PR DESCRIPTION
**Pull request by varkoly**

### CODE REVIEW

**Comment by liangxin1300** [comment]
Hi @varkoly 
Could you please add some background or description about this PR?:)
Thanks!

**Comment by varkoly** [comment]
There is a JIRA request to make HA envinroment for cryptcl: https://jira.suse.com/browse/PED-1326
For documentation see the new created scripts/cryptctl/README.md

File: scripts/cryptctl/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Intorducion
+
+The cryptctl server daemon provides a LUKS-based disk encryption. This script aims to setup an HA environment for the cryptctl-server
+
+## Prerequsits
+
+The cryptctl server needs following resources
+
+* /etc/sysconfig/cryptctl-server The configuration of the server. This will be created once the server was setup und will be only modified if the configuration will be changed like changing the administrator password. It is sufficient to copy this file to all nodes when the cluster will be created.
```
Comment (liangxin1300) [inline scripts/cryptctl/README.md:11]: was setup `and` will be?

File: scripts/cryptctl/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+As first step you have to setup the cryptctl server:
+```shell
+cryptctl init-server
+```
+
+### Create a basic cluster
+If not already done you have to setup a basic cluster with at last two nodes. It is very important that Node1 must be the server where you have confiugred the cryptctl server.
+
+```shell
+crm cluster init -i <NetDev> -A <AdminIP> -n <ClusterName> -N <Node1>
```
Comment (liangxin1300) [inline scripts/cryptctl/README.md:28]: If `Node1` is the local node, here is not necessary to use `-N <Node1>`, and I suggest to add `-y` here

**Comment by liangxin1300** [comment]
> There is a JIRA request to make HA envinroment for cryptcl: https://jira.suse.com/browse/PED-1326 For documentation see the new created scripts/cryptctl/README.md

Hi @varkoly , did you tested and verified this script on your environment?

**Comment by varkoly** [comment]
Yes I've tested it with 2 nodes with nfs share. Setup works fine.

**Review by liangxin1300 — APPROVED** [review]

**Pull request merged**